### ISSUE TITLE
Getting error code 9 on IOS during login

### ISSUE THREAD

**Issue by ryan-1194** [AgoraIO/agora-react-native-rtm#67]
Hi there,

I am having an issue logging in to agora  when using LTE connection on IOS. While searching i found this post https://stackoverflow.com/questions/69095140/agora-rtm-login-getting-a-timeout-error-code-9 on stack overflow, I was just wondering if it is related to this issue?

Thanks.

**Comment by plutoless** [comment]
so it works on other network?

**Comment by ryan-1194** [comment]
Yes it works using wifi with IPv4. But using other network with IPv6 does not work.

**Comment by plutoless** [comment]
@ryan-1194 let me confirm with SDK team. will come back to you soon

**Comment by ryan-1194** [comment]
Thank you so much @plutoless . We only have problem with IOS, android works fine.

**Comment by plutoless** [comment]
@ryan-1194 just confirmed currently version do have issues supporting ipv6. we will release a version supporting ipv6 fully on August.

**Comment by ryan-1194** [comment]
@plutoless Thank you for the confirmation. By the way do you have any work around or suggestions regarding this while waiting for the version that supports ipv6?

**Comment by plutoless** [comment]
@ryan-1194 i'm sorry but there's no known workaround for this for the moment. i'm afraid you will have to wait till august.

**Comment by ryan-1194** [comment]
@plutoless Thank you for the response

**Comment by maxbbb** [comment]
Hi @plutoless, also seeing this issue. For my app we rely on the rtm package for some solutions to live streaming setups as mentioned in the agora docs, this issue not being resolved until August will force us to migrate off of using agora-rtm. Is there any possibility of this issue being fixed sooner?
